### PR TITLE
fix: fix auto size when no content in first col

### PR DESCRIPTION
**Pull request by lumixraku**
close https://github.com/dream-num/univer-pro/issues/3096

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/engine-render/src/components/sheets/sheet-skeleton.ts [hunk 1]
```diff
@@ -1393,7 +1395,7 @@ export class SpreadsheetSkeleton extends Skeleton {
      * @param cell
      * @param options
      */
-    // eslint-disable-next-line complexity
+    // eslint-disable-next-line complexity, max-lines-per-function
```
Comment (wzhudev) [inline packages/engine-render/src/components/sheets/sheet-skeleton.ts:1398]: Do not add this. 😢

File: packages/engine-render/src/components/sheets/sheet-skeleton.ts [hunk 2]
```diff
@@ -689,6 +689,7 @@ export class SpreadsheetSkeleton extends Skeleton {
      * @param colIndex
      * @returns {number} width
      */
+    // eslint-disable-next-line max-lines-per-function
```
Comment (wzhudev) [inline packages/engine-render/src/components/sheets/sheet-skeleton.ts:692]: Consider extract code pieces that can be reused.

**Review by wzhudev — COMMENTED** [review]
Some readability issues need to be addressed.

**Comment by oumomomo** [comment]
无内容也会改动列宽
企业微信截图_17304614534334

**Review by wzhudev — APPROVED** [review]

**Pull request merged**